src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  width: 400px;
+  height: 400px;
```

Comment:
please check task description
size of watch should be 80vmin and in the middle of screen
![Screenshot from 2023-11-11 15-29-31](https://github.com/mate-academy/layout_stop-watch/assets/33029864/12d106b2-fe03-4a8b-9beb-633fd374b988)